vllm/multimodal/utils.py

```diff
@@ -56,9 +56,9 @@ async def fetch_image(cls, image_url: str) -> Image.Image:
         return image
 
 
-async def async_get_and_parse_image(image_url: str) -> ImagePixelData:
+async def async_get_and_parse_image(image_url: str) -> ImageData:
     with await ImageFetchAiohttp.fetch_image(image_url) as image:
-        return ImagePixelData(image)
+        return ImageData(image)
```

Comment:
Can delete this method if we only need the PIL image (just use `fetch_image` directly)

Reply:
I ended up leaving it, it adds a key "image". I don't want the poor API server to have to handle that when we add more modality.

Reply:
What do you mean by that?

Reply:
yeah agree with you, the file is not that organized anyways.
I need to define an async function to return {"image": image} anyways. I moved it to an inline function in `serving_chat.py`.